rgb-matrix.sh

```diff
@@ -7,7 +7,7 @@
 # we reference a specific commit (update this as needed):
 GITUSER=https://github.com/hzeller
 REPO=rpi-rgb-led-matrix
-COMMIT=814b79b5696d32dd1140304b41a1ec0068bb271a
+COMMIT=5549b6a8539f6191b0042a9882844a36716acebd
```

Comment:
Can you change this to the next commit e3dd56dcc0408862f39cccc47c1d9dea1b0fb2d2 ?
I've added more messaging around the fact that the new version also can run as non-root, but better point out the limitations. This will safe me many support emails :D

Reply:
Updated commit as requested.